tests/modules/biscuit/vcf2bed/main.nf

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { BISCUIT_VCF2BED } from '../../../../modules/biscuit/vcf2bed/main.nf'
+
+workflow test_biscuit_vcf2bed {
+
+    input = [
+        [ id:'test', single_end:false ], // meta map
+        file('/varidata/research/projects/laird/nathan/projects/pipelining/test-datasets/data/delete_me/biscuit/test.vcf.gz', checkIfExists: true)
```

Comment:
could you use vcf files from `nf-core/test-datasets` ? @njspix

Reply:
No, these .vcf files have methylation information in them specific to this tool. I just updated this with a GitHub path so tests should be able to run now...